craft_application/launchpad/models/recipe.py

```diff
@@ -285,7 +271,13 @@ def get(  # pyright: ignore[reportIncompatibleMethodOverride]
         try:
             return cls(
                 lp,
-                lp.lp.snaps.getByName(owner=util.get_person_link(owner), name=name),
+                retry(
```

Comment:
Launchpad has been miserably slow today and my builds have been failing half the time on this line.

With your PR, they are passing more often but it's taking until the 4th retry.

What do you think about increasing the retry delay?  (I can't recall what you mentioned this morning about changing the retry backoff)

Reply:
next pr!